p2p/pre2p/stdnetwork/network.go

```diff
@@ -66,3 +68,6 @@ func (n *network) AddPeerToAddrBook(peer *typesPre2P.NetworkPeer) error {
 func (n *network) RemovePeerToAddrBook(peer *typesPre2P.NetworkPeer) error {
 	panic("RemovePeerToAddrBook not implemented")
 }
+
+func (n *network) GetBus() modules.Bus  { return nil }
+func (n *network) SetBus(_ modules.Bus) {}
```

Comment:
Why even make it an integrable module if we don't need this?

Reply:
This is just to comply with the interface as this is struct that implements the `Network` interface and this interface is composed of `IntegrateableModule`, as is the case for `RainTreeNetwork`. It's for compliance sake.